Source/Editor/Viewport/EditorViewport.cs

```diff
@@ -143,6 +144,11 @@ public void Clear()
         private Vector2 _mouseDeltaLast;
         private Vector2[] _deltaFilteringBuffer = new Vector2[FpsCameraFilteringFrames];
 
+        /// <summary>
+        /// Mouse delta (x = Left, Y = Right)
+        /// </summary>
+        protected Vector2 _mouseDelta;
```

Comment:
Where did that come from and is it still used?

Reply:
This I think was some left over stuff from an attempt to fix some dragging issue, its actually duplicated now.

Reply:
Fixed.